src/data/components.json

```diff
@@ -1,124 +1,513 @@
+      }
     },
     {
-      "component": "Loading"
+      "component": "Definition tooltip",
+      "parentComponent": "Tooltip",
+      "overview": false,
+      "testing": {
+        "screenreader": "manual"
+      }
     },
     {
-      "component": "Menu"
+      "component": "Dropdown",
+      "testing": {
+        "screenreader": "manual"
+      }
     },
     {
-      "component": "Menu buttons"
+      "component": "ErrorBoundary",
+      "overview": false,
+      "testing": {
+        "screenreader": "notavailable"
+      }
     },
     {
-      "component": "Modal"
+      "component": "File uploader",
+      "testing": {
+        "screenreader": "manual"
+      }
     },
     {
-      "component": "Multiselect"
+      "component": "FlexGrid",
+      "overview": false,
+      "testing": {
+        "screenreader": "notavailable"
+      }
     },
     {
-      "component": "Notification"
+      "component": "Fluid combo box",
+      "parentComponent": "Dropdown",
+      "overview": false,
+      "testing": {
+        "screenreader": "manual"
+      }
     },
     {
-      "component": "Number input"
+      "component": "Fluid date picker",
+      "parentComponent": "Date picker",
+      "overview": false,
+      "testing": {
+        "screenreader": "manual"
+      }
     },
     {
-      "component": "Pagination"
+      "component": "Fluid dropdown",
+      "parentComponent": "Dropdown",
+      "overview": false,
+      "testing": {
+        "screenreader": "manual"
+      }
     },
     {
-      "component": "Popover"
+      "component": "Fluid form",
+      "parentComponent": "Form",
+      "overview": false,
+      "testing": {
+        "screenreader": "manual"
+      }
     },
     {
-      "component": "Progress bar"
+      "component": "Fluid multiselect",
+      "parentComponent": "Dropdown",
+      "overview": false,
+      "testing": {
+        "screenreader": "manual"
+      }
     },
     {
-      "component": "Progress indicator"
+      "component": "Fluid number input",
+      "parentComponent": "Number input",
+      "overview": false,
+      "testing": {
+        "screenreader": "manual"
+      }
     },
     {
-      "component": "Radio button"
+      "component": "Fluid search",
+      "parentComponent": "Search",
+      "overview": false,
+      "testing": {
+        "screenreader": "manual"
+      }
     },
     {
-      "component": "Search"
+      "component": "Fluid select",
+      "parentComponent": "Select",
+      "overview": false,
+      "testing": {
+        "screenreader": "manual"
+      }
     },
     {
-      "component": "Select"
+      "component": "Fluid textarea",
+      "parentComponent": "Textarea",
+      "overview": false,
+      "testing": {
+        "screenreader": "manual"
+      }
     },
     {
-      "component": "Slider"
+      "component": "Fluid textinput",
+      "parentComponent": "Textinput",
+      "overview": false,
+      "testing": {
+        "screenreader": "manual"
```

Comment:
@andreancardona have our fluid components been screen reader tested?

Reply:
unstable ones no. stable fluid yes!

Reply:
Thanks, updated the data to reflect that. I believe only fluid textinput is stable from what I can see.